src/components/TeamMember/index.tsx

```diff
@@ -1,118 +1,97 @@
 import CloudinaryImage from 'components/CloudinaryImage'
 import Tooltip from 'components/Tooltip'
 import { graphql, useStaticQuery } from 'gatsby'
-import { StaticImage } from 'gatsby-plugin-image'
 import React from 'react'
 import ReactCountryFlag from 'react-country-flag'
 
-export const TeamMemberLink = (person = false) => {
-    const {
-        firstName,
-        lastName,
-        squeakId,
-        avatar,
-        companyRole,
-        location,
-        country,
-        photo = false,
-        color,
-        className,
-        showOnlyFirstName = false,
-    } = person ?? {}
-    let displayName: string | void = ''
-    if (person) {
-        displayName = showOnlyFirstName
-            ? [firstName].filter(Boolean).join(' ')
-            : [firstName, lastName].filter(Boolean).join(' ')
-    } else {
-        displayName = name
-    }
```

Comment:
Not even sure how this got merged in the first place!

Reply:
Yep, me neither